__tests__/schema.ts

```diff
@@ -0,0 +1,33 @@
+import { matchersWithOptions } from 'jest-json-schema';
+
+import accountSchemas from './schemas/account.json';
+import libSchemas from './schemas/lib.json';
+import providerSchemas from './schemas/provider.json';
+import sequencerSchemas from './schemas/sequencer.json';
+
+const ajvKeywords = require('ajv-keywords');
+
+export const initializeMatcher = (expect: jest.Expect) => {
```

Comment:
See if this (or something similar) would work:

```suggestion
const schemas = [...]
const jestJsonMatchers = matchersWithOptions(...)

export const initializeMatcher = (expect: jest.Expect) => {
  expect.extend(jestJsonMatchers)
  expect.extend({
    toMatchSchemaRef(received: object, name: string) {
      const schema = schemas.find((s) => Object.keys(s.definitions).includes(name))
      const $ref = `${schema?.$id}#/definitions/${name}`
      return jestJsonMatchers.toMatchSchema.call(this, received, {$ref})
    },
  })
  ...
declare global {
  namespace jest {
    interface Matchers<R> {
      toMatchSchemaRef(name: string): R;
    }
  }
}
```

It should allow for the usage in the tests to be shorter: `expect(contractClass).toMatchSchema('ContractClass')`